internal/storage/memory/tracer.go

```diff
@@ -0,0 +1,7 @@
+package memory
+
+import (
+	"go.opentelemetry.io/otel"
+)
+
+var tracer = otel.Tracer("storage.postgres")
```

Comment:
It should be `var tracer = otel.Tracer("storage.memory")`. Could you make this change?

Reply:
done